x-pack/plugins/infra/server/lib/host_details/get_services.ts

```diff
@@ -79,12 +79,35 @@ export const getServices = async (
         filter: [
           {
             term: {
-              'metricset.name': 'transaction',
+              'processor.event': 'metric',
```

Comment:
since we're only getting services by metrics in this query I thought i'd add this